hugo/content/downloads.md

```diff
@@ -52,7 +52,7 @@ function getPlatformContent(platform) {
         case 'linux':
             return getButton('ubuntu', 'ubuntu.svg', 'Mumble PPA for Ubuntu')
         case 'macos':
-            return getButton('osx', 'apple.svg', 'Mumble for macOS')
+            return getButton('osx', 'apple.svg', 'Mumble for macOS >= 10.13')
```

Comment:
@Kissaki I did a quick search on the internet but it seems there is no _easy_ way of detecting the actual macOS version. Therefore I just added the note that this is for >= 10.13. If you have the knowledge and motivation to let this script figure out the macOS version, that'd be great.
If not, I think it is also fine the way it currently is though :shrug: 

For <= 10.12 it should return `osx-hfs+` instead of `osx`.

If not